lib/datadog/core/telemetry/event.rb

```diff
@@ -338,27 +338,31 @@ def payload
         class Log < Base
           LEVELS = {
             error: 'ERROR',
-            debug: 'DEBUG',
             warn: 'WARN',
           }.freeze
 
           def type
             'logs'
           end
 
-          def initialize(message:, level:)
+          def initialize(message:, level:, stack_trace: nil)
             super()
             @message = message
+            @stack_trace = stack_trace
             @level = LEVELS.fetch(level) { |k| raise ArgumentError, "Invalid log level :#{k}" }
           end
 
           def payload
             {
-              logs: [{
-                message: @message,
-                level: @level,
-                # More optional fields to be added here...
-              }]
+              logs: [
+                {
+                  # Required fields
+                  message: @message,
+                  level: @level,
+                  # More optional fields to be added here...
```

Comment:
Should we perhaps add a message about consulting some documentation / with some team before adding elements here? E.g. to avoid inconsistencies with other libraries.

Reply:
Ping? :arrow_up: